Lib/fontTools/varLib/varStore.py

```diff
@@ -551,8 +551,16 @@ def VarStore_optimize(self, use_NO_VARIATION_INDEX=True):
 
         for minor, item in enumerate(data.Item):
             row = list(zeroes)
-            for regionIdx, v in zip(regionIndices, item):
-                row[regionIdx] += v
+
+            if quantization == 1:
+                for regionIdx, v in zip(regionIndices, item):
+                    row[regionIdx] += v
+            else:
+                for regionIdx, v in zip(regionIndices, item):
+                    row[regionIdx] += (
+                        round(v / quantization) * quantization
+                    )  # TODO: round towards 0
```

Comment:
why "round towards 0"?

you can use `decimal.Decimal`'s `quantize` method with a `ROUND_DOWN` rounding method:

https://docs.python.org/3/library/decimal.html#decimal.ROUND_DOWN

though it might be quicker to simply do `floor(v) if v > 0 else ceil(v)`

Reply:
I'll remove the comment. round is fine.

Reply:
What I meant was a round that rounds .5 to 0, such that eg. with quantization 2, value 1 will become 0, not 2.

Reply:
ah that's https://docs.python.org/3/library/decimal.html#decimal.ROUND_HALF_DOWN

Reply:
yeah i think as a size saving optimization this quantization should make the deltas smaller, would be strange if it made them bigger

Reply:
(though the sharing comes from reuse, not the delta magnitude itself)